src/components/NavbarComponent.js

```diff
@@ -12,10 +12,12 @@ import {
  * @property {HTMLElement} components.openNew button to click for upload a new pdf from the reader component
  * @property {HTMLElement} components.loader loader for showing the pdf uploaded
  * @property {HTMLElement} components.errorMessage error message for a wrong pdf uploaded from the pdf reader component
+ * @property {HTMLElement} components.highlightToggle toggle button for turning on/off highlighted sentences
  */
 class NavbarComponent {
   components = {
     openNew: document.querySelector("#open-new-pdf"),
+    highlightToggle: document.querySelector("#highlight-toggle"),
```

Comment:
NavbarComponent is deleted in refactoring task since it was initially created to only hold "upload new pdf file" button, but all the other buttons in navigation bar is in ToolbarComponent. I would highly recommend to not use this component, and move this logic to ToolbarComponent.

Reply:
What refactoring task?